ddtrace/internal/sampling.py

```diff
@@ -166,3 +175,67 @@ def apply_span_sampling_tags(self, span):
         span.set_metric(_SINGLE_SPAN_SAMPLING_RATE, self._sample_rate)
         if self._max_per_second:
             span.set_metric(_SINGLE_SPAN_SAMPLING_MAX_PER_SEC, self._max_per_second)
+
+
+class SpanSamplingRules:
+    """To generate a list of SpanSamplingRule objects and pass them to the SpanProcessor"""
+
+    def __init__(self):
+        self._rules = self._make_rules()
+
+    def get_single_span_sampling_rules(self):
+        # type: () -> List[SpanSamplingRule]
+        return self._rules
+
+    def _make_rules(self):
+        # type: () -> List[SpanSamplingRule]
+        json_rules = os.getenv("DD_SPAN_SAMPLING_RULES")
```

Comment:
```suggestion
        json_rules = os.getenv("DD_TRACER_SPAN_SAMPLING_RULES")
```
?

Reply:
No, the name is correct. I also started writing that name before reviewing it though.

Reply:
I wonder if this shouldn't be changed though 🤔 . In light of the effort of creating shared configuration, where would this variable stand? I'd say this is a tracer-related setting, so to me it would make sense to add the `DD_TRACER` prefix. WDYT?

Reply:
I'm not strongly opinionated, but I know other tracers have already finished work using `DD_SPAN_SAMPLING_RULES`. I'll ask the RFC owner and see what they think.

Reply:
It seems the RFC owner would like to keep it DD_SPAN_SAMPLING_RULES